biogrid/libs/biogrid-simulator/src/lib/biogrid-state/biogrid-state.spec.ts

```diff
@@ -1,35 +1,38 @@
 import { BiogridState } from './';
-import { NewStateGraphVertex } from '@biogrid/grid-simulator';
+import { StateGraphVertex } from '@biogrid/grid-simulator';
+import { Building } from '../building';
 
 describe('classes', () => {
+  const x1 = 10,
+    y1 = 5;
+  const x2 = 7,
+    y2 = 9;
   it('should work to create a BiogridState', () => {
-    const newVertices: NewStateGraphVertex[] = [
-      {
-        vertex: 'v 1',
-        edges: [
-          {
-            toIndex: 1,
-            weight: 10,
-            label: 'v 1 label',
-          },
-        ],
-      },
-      {
-        vertex: 'v 2',
-        edges: [
-          {
-            toIndex: 0,
-            weight: 11,
-            label: 'v 2 label',
-          },
-        ],
-      },
-    ] as NewStateGraphVertex[];
-    const state = new BiogridState(2, newVertices);
+    const newVertices: StateGraphVertex[] = [
+      new Building(32, x1, y1),
```

Comment:
when creating objects like this in tests, it can often be confusing to a reader to know what the parameters are without having to look up the Building class. Usually variable names at least give us an idea of what the parameter is supposed to be (e.g. x1 and y1 make sense), but for non-variable inputs (such as the 32) it can help to do an inline comment for readability: 
`new Building( /* distance= */ 32, x1, y1)`